sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/ecert-part-time-process-integration.scheduler.e2e-spec.ts

```diff
@@ -328,6 +328,120 @@ describe(
       expect(scheduleIsSent).toBe(true);
     });
 
+    it("Should adjust tuition remittance when requested tuition remittance is greater than the max tuition remittance.", async () => {
+      // Arrange
+
+      // Student with valid SIN.
+      const student = await saveFakeStudent(db.dataSource);
+      // Valid MSFAA Number.
+      const msfaaNumber = await db.msfaaNumber.save(
+        createFakeMSFAANumber(
+          { student },
+          {
+            msfaaState: MSFAAStates.Signed,
+            msfaaInitialValues: {
+              offeringIntensity: OfferingIntensity.partTime,
+            },
+          },
+        ),
+      );
+      // Student application eligible for e-Cert.
+      const application = await saveFakeApplicationDisbursements(
+        db.dataSource,
+        {
+          student,
+          msfaaNumber,
+          disbursementValues: [
+            createFakeDisbursementValue(
+              DisbursementValueType.CanadaLoan,
+              "CSLP",
+              1000,
+            ),
+          ],
+          secondDisbursementValues: [
+            createFakeDisbursementValue(
+              DisbursementValueType.CanadaLoan,
+              "CSLP",
+              1000,
+            ),
+          ],
+        },
+        {
+          offeringIntensity: OfferingIntensity.partTime,
+          applicationStatus: ApplicationStatus.Completed,
+          createSecondDisbursement: true,
+          currentAssessmentInitialValues: {
+            assessmentData: { weeks: 5 } as Assessment,
+            assessmentDate: new Date(),
+          },
+          firstDisbursementInitialValues: {
+            coeStatus: COEStatus.completed,
+            tuitionRemittanceRequestedAmount: 500,
+            tuitionRemittanceEffectiveAmount: 500,
+            dateSent: new Date(),
+            readyToSendDate: new Date(),
+            disbursementScheduleStatus: DisbursementScheduleStatus.Sent,
+          },
+          secondDisbursementInitialValues: {
+            coeStatus: COEStatus.completed,
+            tuitionRemittanceRequestedAmount: 700,
+            readyToSendDate: new Date(),
+            disbursementDate: getISODateOnlyString(new Date()),
+          },
+        },
+      );
+      // Adjust offering values for maxTuitionRemittanceAllowed.
+      application.currentAssessment.offering.actualTuitionCosts = 500;
+      application.currentAssessment.offering.programRelatedCosts = 500;
```

Comment:
As a suggestion, having different values associated here would make it easier to understand which one was picked.